impl/src/test/java/org/ehcache/impl/config/serializer/SerializerCountingTest.java

```diff
@@ -92,7 +91,7 @@ public void testOnHeapPutGet() {
     printSerializationCounters("Get OnHeap");
 
     cache.put(42L, "Wrong ...");
-    assertCounters(2, 2, 0, 1, 0, 0);
+    assertCounters(2, 2, 0, 1, 1, 0);
```

Comment:
Put that underneath uses compute does indeed add extra deserialization of value. Adding a general comment to attract attention.